civiform/civiform
# Import USWDS component library

### Description

This PR pulls in the [U.S Web Design System (USWDS)](https://designsystem.digital.gov/about/) component library.  Once this library is available to us, we will be able to easily use any USWDS out-of-the-box components or styles.  With this PR, the only style change is the color and thickness of the focus box. 

## Release notes

This PR pulls in the [U.S Web Design System (USWDS)](https://designsystem.digital.gov/about/) component library.  Once this library is available to us, we will be able to easily use any USWDS out-of-the-box components or styles. 

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1.  Open the developer tools in your browser.
2. Go to the Network tab.
3. Load the CiviForm app.
4. Note the presence of `uswds.min.css`, `uswds.bundle.js` and `uswds-init.min.js`
5. Also, use the tab key and notice the new focus styles:
![image](https://github.com/civiform/civiform/assets/134556723/bc9a239d-9833-411f-9a74-51e7ff31640f)


### Issue(s) this completes

Fixes #5662; #5886
